WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaItem.kt

```diff
@@ -17,20 +18,61 @@ data class MediaItem(
         data class LocalUri(val value: UriWrapper) : Identifier()
         data class RemoteId(val value: Long) : Identifier()
         data class StockMediaIdentifier(val url: String?, val name: String?, val title: String?) : Identifier()
+        data class GifMediaIdentifier(
```

Comment:
I think this doesn't need all these fields, right? Only the fields necessary in the Insert use case need to be part of the identifier, are the other fields ever read from this class? It could be only `largeImageUri` and the `title`.

Reply:
I left it like this after originally mapping the fields from `data class MutableGifMediaViewModel` and then forgot to simplify 🤦‍♂️! Changed in 546cc427e28d0554813d57c93a7ffd6de24a1817 👍